client/signup/main.jsx

```diff
@@ -118,6 +118,7 @@ class Signup extends React.Component {
 		pageTitle: PropTypes.string,
 		siteType: PropTypes.string,
 		stepSectionName: PropTypes.string,
+		shouldShowMockup: PropTypes.bool,
```

Comment:
Nitpick: could this prop match the plural-ness of `showSiteMockups`? Would like to avoid a proliferation of spellings 😅

Reply:
Good call. 🙇